core/templates/dev/head/pages/admin/roles_tab/roles_tab_directive.html

```diff
@@ -0,0 +1,98 @@
+<style>
+    .form-horizontal .control-label {
```

Comment:
Deindent this and the next two lines by 2.

Also, shouldn't we add something at the front (maybe `roles-tab`) to scope this to only the roles-tab directive?